### ISSUE TITLE
make new repo with hybrid version

### ISSUE THREAD

**Issue by ArtskydJ** [ArtskydJ/convert-and-seed-audio#7]
make new repo with hybrid version, so wrtc does not install on this version

```js
global.WRTC = require('wrtc')
global.WEBTORRENT_ANNOUNCE = [ 'wss://tracker.webtorrent.io' ]
```

**Comment by ArtskydJ** [comment]
Actually just connect to web peers in this module.

**Comment by ArtskydJ** [comment]
Fixed in 27ebcbe1b6440f70c2d9e0501b74d1a29090f05a.
